spa/src/elements/systemNotification/SystemNotification.tsx

```diff
@@ -0,0 +1,52 @@
+import { IconBox, IconBoxIcon, IconButton, Header, Main, SystemNotificationWrapper } from './SystemNotification.styled';
+import {
+  CheckCircleOutlineOutlined as SuccessIcon,
+  CloseOutlined as CloseIcon,
+  ErrorOutlineOutlined as ErrorIcon,
+  InfoOutlined as InfoIcon,
+  ReportProblemOutlined as WarningIcon
+} from '@material-ui/icons';
+
+import { SystemNotificationTypes } from './commonTypes';
+
+interface SystemNotificationProps {
+  type: SystemNotificationTypes;
+  header: string;
+  body: string;
+  handleClose: Function;
```

Comment:
```suggestion
  handleClose: () => void;
```

Prevents you from using the wrong function signature.

Reply:
630271cb7d